Releases.md

```diff
@@ -19,6 +20,11 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 | [1.1.0](#directml-110) | [DML_FEATURE_LEVEL_2_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_2_0) | Version 2004 (Build 10.0.19041; 20H1, "May 2020 Update") | -                                                                                         |
 | [1.0.0](#directml-100) | [DML_FEATURE_LEVEL_1_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_1_0) | Version 1903 (Build 10.0.18362; 19H1, "May 2019 Update") | -                                                                                         |
 
+# DirectML 1.9.1
+- UINT64 support for OneHot operator.
```

Comment:
ONE_HOT has supported UINT64 ever since feature level 3_0. This would be better described as a fix for an issue that prevented certain index values from working correctly, when used with UINT64 tensors.

Reply:
Make sense. This actually added uint64 support for **IndicesTensor** in OneHot operator. I will update that.